vendor/angle-bracket-invocation-polyfill/runtime-polyfill.js

```diff
@@ -245,176 +407,16 @@ import { lte, gte } from 'ember-compatibility-helpers';
-                  references,
-                  dynamicAttributes,
-                  dom,
-                  environment,
-                };
-              },
-
-              getTag({ references }) {
-                return combineTagged(references);
-              },
-
-              install(bucket) {
-                let { dynamicAttributes } = bucket;
-
-                for (let name in dynamicAttributes) {
-                  let attribute = dynamicAttributes[name];
-                  attribute.set();
-                }
-              },
-
-              update(bucket) {
-                let { dynamicAttributes } = bucket;
-
-                for (let name in dynamicAttributes) {
-                  let attribute = dynamicAttributes[name];
-                  attribute.update();
-                }
-              },
-
-              getDestructor() {},
-            };
-
-            let originalGetComponentDefinition = environment.getComponentDefinition;
-            environment.getComponentDefinition = function() {
-              let definition = originalGetComponentDefinition.apply(this, arguments);
-
-              if (!installedCustomDidCreateElement && definition) {
-                installedCustomDidCreateElement = true;
-
-                let { manager } = definition;
-
-                let ORIGINAL_DID_CREATE_ELEMENT = manager.didCreateElement;
-                manager.didCreateElement = function(bucket, element, operations) {
-                  ORIGINAL_DID_CREATE_ELEMENT.apply(this, arguments);
-                  let { args } = bucket;
-
-                  if (lte('2.15.0-beta.1')) {
-                    args = args.namedArgs;
-                  }
-
-                  // on < 2.15 `namedArgs` is only present when there were arguments
-                  if (args && args.has('__ANGLE_ATTRS__')) {
-                    let attributeReferences = args.get('__ANGLE_ATTRS__');
-                    let snapshot = attributeReferences.value();
-                    if (snapshot) {
-                      let names = Object.keys(snapshot);
-                      for (let i = 0; i < names.length; i++) {
-                        let attributeName = names[i];
-                        let attributeReference = attributeReferences.get(attributeName);
-
-                        operations.addDynamicAttribute(
-                          element,
-                          attributeName,
-                          attributeReference,
-                          false,
-                          null
-                        );
-                      }
-                    }
-                  }
-                };
-              }
-
-              return definition;
-            };
-
-            return environment;
-          };
-          Environment.create.__IS_ANGLE_BRACKET_PATCHED__ = true;
-        }
-
-        return registry;
-      },
-    });
+      return registry;
+    };
+    for (const PolyfillTarget of [Application, Engine]) {
```

Comment:
Sadly, I don't think we can use `for of` loops (because it ends up requiring `Symbol` which is not available on IE11), would you mind rewriting as either a plain `for` or `.forEach`?

Reply:
Done... rather than replace it with a different loop I just unrolled it, since it was only two elements.